Silence MSVC Windows XP deprecation warning (

With VS2017 and XP-compatible builds enabled we have these warnings:
```
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\Platforms\x64\PlatformToolsets\v141_xp\Toolset.targets(39,5): warning MSB8051: Support for targeting Windows XP is deprecated and will not be present in future releases of Visual Studio. Please see https://go.microsoft.com/fwlink/?linkid=2023588 for more information. [C:\projects\radare2\build\shlr\cafa177@@r2qnx@sta.vcxproj]
```
They are quite annoying and clogging the useful log output, see example here https://ci.appveyor.com/project/radareorg/radare2/builds/29572763/job/h13vb3lf64fef3b6?fullLog=true

It would be amazing to disable it 

See https://stackoverflow.com/questions/53841470/how-to-disable-warning-msb8051-support-for-targeting-windows-xp-is-deprecated

--- a/sys/meson.py
+++ b/sys/meson.py
@@ -233,7 +233,10 @@ def build(args):
             xp_compat(r2_builddir)
         if not args.project:
             project = os.path.join(r2_builddir, 'radare2.sln')
-            msbuild(project, '/m')
+            params = ['/m']
+            if args.backend == 'vs2017' and args.xp:
+                params.append('/p:XPDeprecationWarning=false')
+            msbuild(project, *params)
     else:
         ninja(r2_builddir)
 

